pkg/operation/botanist/wanted_secrets.go

```diff
@@ -679,6 +680,42 @@ func (b *Botanist) generateWantedSecretConfigs(basicAuthAPIServer *secrets.Basic
 					SigningCA: certificateAuthorities[konnectivity.SecretNameServerCA],
 				})
 		}
+	} else if b.Shoot.ReversedVPNEnabled {
+		secretList = append(secretList,
+			// Secret definition for vpn-shoot (OpenVPN client side)
+			&secrets.CertificateSecretConfig{
+				Name:       "vpn-shoot-client",
+				CommonName: "vpn-shoot-client",
```

Comment:
As the common name of the certificate is used as a file name for the open vpn server in the client configuration directory, I would prefer to not include a ':' character. Having files with special characters is usually a hassle.
What do you prefer (in order of my preference)?
- vpn-shoot-client
- gardener-cloud-system-vpn-shoot-client
- gardener.cloud.system.vpn-shoot-client
- gardener.cloud:system:vpn-shoot-client
- gardener.cloud:system:vpn:vpn-shoot-client

I would rather stay with vpn-shoot-client, but I am open if you feel one of the other names is better.

Reply:
Personally, I would vote for `gardener.cloud:system:vpn-shoot-client`, mainly because this is what we already do for many other components (e.g., Prometheus, etc.). I cannot remember that we ever had issues with the `:`.

Reply:
In general, I do not think that `:` characters cause issues in certificates. However, this common name is (and must be) used as a file name as well. If you think that having a `:` in a file name is a good idea I can make the necessary changes as this also affects the vpn2 repository.
Your Call.

Reply:
OK, @ScheererJ explained via Slack that OpenVPN is evaluating the common name:

```
–client-config-dir dir
Specify a directory for custom client config files. After a connecting client has been authenticated, OpenVPN will look in this directory for a file having the same name as the client's X509 common name. If a matching file exists, it will be opened and parsed for client-specific configuration options. If no matching file is found, OpenVPN will instead try to open and parse a default file called "DEFAULT", which may be provided but is not required.This file can specify a fixed IP address for a given client using –ifconfig-push, as well as fixed subnets owned by the client using –iroute.
```

Under these circumstances, let's keep it for now as is.